libs/react-helpers/src/lib/grid-cells/price-cell.tsx

```diff
@@ -6,12 +6,11 @@ export interface IPriceCellProps {
 }
 
 export const PriceCell = ({ value, valueFormatted }: IPriceCellProps) => {
-  if (!value || isNaN(Number(value))) return <span>-</span>;
+  if ((!value && value !== 0) || isNaN(Number(value)))
```

Comment:
Another code style/consistency question here 😉, but I'd prefer to always have curly braces and not rely in JS indentation